genkit-tools/cli/src/commands/init-ai-tools/command.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { logger } from '@genkit-ai/tools-common/utils';
+import { checkbox } from '@inquirer/prompts';
+import * as clc from 'colorette';
+import { Command } from 'commander';
+import { AI_TOOLS } from './constants';
+import { AIToolChoice, InitConfigOptions } from './types';
+
+const AGENT_CHOICES: AIToolChoice[] = Object.values(AI_TOOLS).map((tool) => ({
+  value: tool.name,
+  name: tool.displayName,
+  checked: false,
+}));
+
+/**
+ * Initializes selected AI tools with Genkit MCP server and Genkit framework
+ * context to improve output quality when using those tools.
+ */
+export const initAiTools = new Command('init:ai-tools')
+  .description(
+    'initialize AI tools in a workspace with helpful context related to the Genkit framework'
+  )
+  .option('-y', '--yes', 'Run in non-interactive mode (experimental)')
```

Comment:
why is this flag experimental? I think the whole command should be experimental to start.